packages/portalnetwork/docs/modules.md

```diff
@@ -547,7 +547,7 @@ Takes an RLP encoded block as a hex string and adds the block header and block b
 | Name | Type | Description |
 | :------ | :------ | :------ |
 | `rlpHex` | `string` | RLP encoded block as hex string |
-| `blockHash` | `string` | block hash as 0x prefixed hext string |
+| `blockHash` | `string` | block hash as 0x prefixed hex string |
```

Comment:
These doc files are autogenerated.  Better to fix the spelling in the original typedoc associated with this function and it will get revised the next time we regenerate the docs.